### PR TITLE
Updated to current Laravel and PHP Versions

### PR DESCRIPTION
**Pull request by Funfare**
Hi,
I updated the code to current supported PHP and Laravel versions.
Also I updated change of rocket chat api (roomId -> channel)

Even if not merged to laravel-notification-channel, can you release the current version? think would be helpful for people even if not in laravel-notification-channel project

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
         }
     ],
     "require": {
-        "php": ">=5.6.4",
+        "php": ">=7.2.0",
         "guzzlehttp/guzzle": "^6.2",
-        "illuminate/notifications": "5.3.*|5.4.*",
-        "illuminate/queue": "5.1.*|5.2.*|5.3.*|5.4.*",
-        "illuminate/support": "5.1.*|5.2.*|5.3.*|5.4.*"
+        "illuminate/notifications": ">5.5.0|^6.0",
```
Comment (antonkomarev) [inline composer.json]: `>5.5.0` is not valid because it will fit 7.0 too. Better replace it with `^5.5`.

Comment (Funfare) [reply composer.json]: yeah, you're right. I changed it

**Comment by antonkomarev** [comment]
Thanks for the PR @FunfareNico!
Do we need to replace chat_id to channel in README file?

**Comment by Funfare** [comment]
think not, i changed only naming in the calls to rocketchat.

**Pull request merged**